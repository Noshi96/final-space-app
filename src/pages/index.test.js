/* globals describe, it, expect  */
/* eslint-env jest */

import { render, waitFor } from '@testing-library/react'
import Home from './index.page'

describe('Home', () => {
  it('should render Home page', async () => {
    const { container } = render(<Home />)
    await waitFor(() => {
      expect(container).toMatchSnapshot()
    })
  })
})
