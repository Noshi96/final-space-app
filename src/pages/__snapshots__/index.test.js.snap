// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render Home page 1`] = `
<div>
  <header
    class="sc-75d2f2ff-0 fbiUAK"
  >
    <div
      class="sc-75d2f2ff-1 gMhXdr"
    >
      <a
        class="sc-75d2f2ff-2 QLpgX"
        href="/"
      >
        <img
          alt="final-space-text"
          class="sc-75d2f2ff-3 fcosyw"
          src="https://images.fanart.tv/fanart/final-space-5e6df59bd745e.png"
        />
      </a>
      <nav>
        <div
          class="sc-e5d9fcd-0 ccJOpB"
        >
          <div />
          <div />
          <div />
        </div>
        <ul
          class="sc-3413b2c7-0 kWOcCk"
        >
          <li>
            <a
              class="sc-3413b2c7-1 cPDNgh"
              href="/characters"
            >
              Characters
            </a>
          </li>
          <li>
            <a
              class="sc-3413b2c7-1 cPDNgh"
              href="/characters"
            >
              Episodes
            </a>
          </li>
          <li>
            <a
              class="sc-3413b2c7-1 cPDNgh"
              href="/characters"
            >
              Info
            </a>
          </li>
          <li>
            <a
              class="sc-3413b2c7-1 cPDNgh"
              href="/characters"
            >
              Games
            </a>
          </li>
        </ul>
      </nav>
    </div>
  </header>
  <main>
    <section
      class="sc-d7ef5778-0 hJbPwZ"
    >
      <div
        class="sc-d7ef5778-1 fWHLxd"
      >
        <div
          class="sc-d7ef5778-2 hyAtHl"
        >
          <h1
            class="sc-d7ef5778-3 iLSkHc"
          >
            Check out your 
            <span>
              favorite
            </span>
             heroes from series and play
             
            <span>
              games
            </span>
          </h1>
          <p
            class="sc-d7ef5778-4 cFOEaL"
          >
            Learn more about 
            <span>
              universum
            </span>
          </p>
          <button
            class="sc-d7ef5778-5 dvncD"
          >
            Click here
          </button>
        </div>
        <img
          alt="final-space-wallpaper"
          class="sc-d7ef5778-6 dhGlcp"
          src="https://images5.alphacoders.com/105/1055769.jpg"
        />
      </div>
    </section>
  </main>
</div>
`;
